linode/manager
# refactor: [M3-8240] - Allow URL path to open OBJ create access key drawer

## Description 📝
Currently, the "create" actions in Cloud Manager include the term 'create' within the URL. Unifying the URL syntax across all creation flows will enhance analytics segmentation.

## Changes  🔄
- Access Key Creating has `/create` in URL Path

## Target release date 🗓️
N/A

## Preview 📷


| Screenshots  |
| ------- | 
| ![Screenshot 2024-08-05 at 3 14 45 PM](https://github.com/user-attachments/assets/c71e54b9-8548-4f9f-ad8d-8d7034d18e47) |

## How to test 🧪

### Prerequisites
- Check with OBJ Gen2 and multicluster Flags on/off

### Verification steps
- Try creating access key and observe URL has /create
- Ensure all these open when typing directly into address bar

## As an Author I have considered 🤔

*Check all that apply*
